e2e-tests/commands/api/index.js

```diff
@@ -162,7 +162,7 @@ const queryStrings = {
       }
     `,
     getLatestHowDataWillBeUsed: () => gql`
-      query declarationHowDataWillBeUsed {
+      query declarationHowDataWillBeUseds {
```

Comment:
Should it not be `declarationHowDataWillBeUsed ` and not `declarationHowDataWillBeUseds `?

Reply:
It is strange, but this is correct. All declarations are fetched essentially by a `findAll`. The Keystone GQL generates this available GQL query by adding an `s` to the entity type. E.g this could be `policyAndExports`. 

This is how all the other declarations are, just this declaration had an incorrect name.